src/bun.js/bindings/JSPropertyIterator.cpp

```diff
@@ -37,6 +38,7 @@ extern "C" JSPropertyIterator* Bun__JSPropertyIterator__create(JSC::JSGlobalObje
     JSC::VM& vm = globalObject->vm();
     JSC::JSValue value = JSValue::decode(encodedValue);
     JSC::JSObject* object = value.getObject();
+    ASSERT(object != NULL);
```

Comment:
This assertion doesn't do anything, getObject() will crash before this line is reached if that assertion wasn't true

Reply:
The implementation of getObject() is this which returns a null pointer
```cpp
inline JSObject* JSValue::getObject() const
{
    return isCell() ? asCell()->getObject() : nullptr;
}
```

Reply:
isCell() when given a null pointer returns true (yes, surprising)

asCell()->getObject() then de-references the null pointer and 💥